ProcessOptimizer/space/space.py

```diff
@@ -313,6 +317,44 @@ def lhs_arange(self, n):
         # Transform the samples to the range used for this dimension
         return samples*(self.high - self.low) + self.low
 
+    def evenly_sample(self, num_points, border_width=None):
+        """Return `n_points` evenly spaced points from a Dimension.
+
+        Parameters
+        ----------
+        * `num_points` [int]
+            The number of points to sample.
+
+        * `border_width` [int]
+            How wide to make the borders. If None, no borders are returned.
+
+        Returns
+        -------
+        * `xi`: [np.array]: The sampled points.
+
+        * `xi_transformed`: [np.array]:
+            The transformed values of `xi`, for feeding to a model.
+
+        * `border_list`: [list of tuples]:
+            A list of where 
+        """
+        bounds = self.bounds
+        xi = np.linspace(bounds[0], bounds[1], num_points)
+        xi_transformed = self.transform(xi)
+        if border_width is None:
+            return xi, xi_transformed
+        border_list = []
+        transformed_bounds = self.transform(self.bounds)
+        border_distance = float(

```

Comment:
From this point and the next lines down, could use adherence to PEP standard (last ")" that closes float onto its own line, spaces around + and -).